src/gamma_inc.jl

```diff
@@ -919,6 +919,17 @@ See also: [`gamma_inc(a,x,ind)`](@ref SpecialFunctions.gamma_inc).
 gamma_inc_inv(a::Real, p::Real, q::Real) = _gamma_inc_inv(promote(float(a), float(p), float(q))...)
 
 function _gamma_inc_inv(a::Float64, p::Float64, q::Float64)
+
+    if p + q != 1
+        throw(ArgumentError("p + q must equal one but is $(p + q)"))
+    end
+
+    if iszero(p)
+        return 0.0
+    elseif iszero(q)
+        return Inf
+    end
```

Comment:
AFAICT this is not part of the Fortran implementation but it seems reasonable.

Reply:
Correct. I compiled the Fortran source and called it from Julia to verify that the end points and handled.